### ISSUE TITLE
i don't know why all my tests that uses selectChoose are broken now ...

### ISSUE THREAD

**Issue by BnitoBzh** [cibernox/ember-power-select#1038]
8 days ago i have updated my application and deployed it with ember-power-select 1.10.4 (with the test-support depreciation) All of my tests worked great !)
But now, and i really don't know why, when i test my application, all selectChoose calls fail ...

I have made a lot of tests during this day : 

change node version 6.10, 6.11, 6.10, 8 ... no effect
relaunch the last succed Travis build (fails now .. grr)
change/force  a specific ember-power-select version ... no effect

i have debugged the selectChoose function, and my analyse is that the dropdown content is not generated, why ? i don't know ...

``let options = findAll(`#${contentId} .ember-power-select-option`);``
`options` is empty, the previous click call (force dropdown opening) has no effect ..

My application works perfectly on Chrome,Firefox, Safari ... only the tests fail ...

Tomorrow i will test with an empty new application ..

Encountered errors come from the error throws from `selectChoose` function at line 128 and from the function `querySelectorAll` due to the call launchs from the line 114 ``potentialTargets = findAll(`#${contentId} ${valueOrSelector}`);`` because `valueOrSelector` is just an option value in my tests

**Comment by sethphillips** [comment]
this is also happening with v `~1.9.6`

**Comment by flexyford** [comment]
Same for me on `1.9.11`

**Comment by danielleadams** [comment]
+1

**Comment by danielleadams** [comment]
+1

I'm using `1.8.5`

FYI: I have a temporary workaround to use shrinkwrap for setting the version of `ember-native-dom-helpers`. It looks like there was a minor version change in the last few days the acceptance test helpers.

I added this and it fixed my tests (I think the breaking dependency is `ember-native-dom-helpers@0.5.9`)
```javascript
    "ember-power-select": {
      "version": "1.8.5",
      "dependencies": {
        "ember-basic-dropdown": {
          "version": "0.32.9",
          "dependencies": {
            "ember-native-dom-helpers": {
              "version": "0.5.8"
            }
          }
        }
      }
    }
  }
```

**Comment by BnitoBzh** [comment]
Ok guys, thank you so much !

Yesterday, my tests showed me that the module XX seemed to be in question, so you confirm my thought.

Today, i have try my test with a new npm install using `npm shrinkwrap` and the options below 
```
"ember-power-select": {
  "version": "1.10.4",
  "dependencies": {
      "ember-basic-dropdown": {
	  "version": "0.34.0",
	  "dependencies": {
		"ember-native-dom-helpers": {
		  "version": "0.5.8"
		}
	  }
      }
  }
}
```
So `ember-native-dom-helpers` version 0.5.9 is the source of the issue !

**Comment by BillyRayPreachersSon** [comment]
See https://github.com/cibernox/ember-native-dom-helpers/pull/121#issuecomment-349978106 and https://github.com/cibernox/ember-native-dom-helpers/pull/121#issuecomment-349998262

**Comment by cibernox** [comment]
I have a fix for this, but it might take a moment because I have to upstream a fix in ember-wormhole, so I'm going to revert the change in ember-native-dom-helpers that causes this problem until ember-wormhole if fixed.

**Comment by danielleadams** [comment]
great. thanks, @cibernox!

**Comment by cibernox** [comment]
If all of you with problems update to 0.6.1 of `ember-native-dom-helpers` (if you don't directly depend on it, just nombom node_modules and install again). It should be fixed.

I'll make sure ember-power-* projects work before updating `ember-native-dom-helpers` again.

**Comment by cibernox** [comment]
I'm going to close as I think it should be fixed if people nombom their dependencies.

**Comment by sethphillips** [comment]
version ~1.9.6 still pulls ember-native-dom helpers <0.6.0 so the fix in 0.6.1 does not apply

**Comment by cibernox** [comment]
@sethphillips If you are using 0.5.X this error shouldn't be there to begin with, since it's was due to a breaking change in 0.6.0. Perhaps I'm missing something.

**Comment by sethphillips** [comment]
i am definitely effected by it and using 1.9.X

**Comment by flexyford** [comment]
Yeah I'm using `yarn` and my `ember-native-dom-helpers` version resolved to `0.5.9` even after a `yarn cache clean` and `yarn upgrade`

``` js
// start-app
import registerPowerSelectHelpers from 'ember-power-select/test-support/helpers';
import { settings } from 'ember-native-dom-helpers';

registerPowerSelectHelpers();

settings.rootElement = '#ember-testing';

export default function startApp(attrs) { /*   */});
```

**Comment by cibernox** [comment]
@sethphillips @flexyford 0.5.9 should have been unpublised. IDK why it's still available. I published 0.5.10 fixing this, so those using 0.5.X will get the fix.

**Comment by sethphillips** [comment]
@cibernox 👍 thank you so much for fast action and a fantastic plugin that we use like 'franks red hot'
